aws/PAS-AIO-DR-template.yaml

```diff
@@ -0,0 +1,1227 @@
+          services:
+            windows:
+              AmazonSSMAgent:
+                enabled: 'true'
+                ensureRunning: 'true'
+                files:
+                  - C:\Program Files\Amazon\SSM\Plugins\awsCloudWatch\AWS.EC2.Windows.CloudWatch.json
+          commands:
+            1-configcw:
+              command: !Sub >
+                powershell.exe -File C:\CyberArk\Deployment\CloudWatch.ps1 -LogGroup ${LogGroup} -CfnInitLogStream ${VaultCfnInitLogStream} -VaultInitLogStream ${VaultInitLogStream} -VaultPostInstallLogStream ${VaultPostInstallLogStream} -Region ${AWS::Region}
+              waitAfterCompletion: '10'
+            2-restartSSM:
+              command: powershell.exe -Command "Restart-Service AmazonSSMAgent"
+              waitAfterCompletion: '60'
+        configServices:
+          commands:
+            1-download:
+              command: !Sub >
+                powershell.exe -File C:\CyberArk\Deployment\VaultInit.ps1 -VaultFilesBucket ${VaultFilesBucket} -LicenseFileKey ${LicenseFile} -RecoveryPublicKey ${RecoveryPublicKey} -Region ${AWS::Region}
+              waitAfterCompletion: '0'
+            2-fixENE:
+              command: >
+                powershell.exe -File C:\CyberArk\Deployment\FixENE.ps1
+              waitAfterCompletion: 0
+            3-postInstall:
+              command: !Sub >
+                powershell.exe -File C:\CyberArk\Deployment\VaultPostInstall.ps1 -SSMMasterPassParameterID ${StoreMasterPassword.SsmId} -SSMAdminPassParameterID ${StoreAdminPassword.SsmId} -IsPrimaryOrDR "Primary" -PrimaryVaultIP "1.1.1.1" -LicensePath "C:\CyberArk\Deployment\vaultLicense.xml" -RecoveryPublicKeyPath "C:\CyberArk\Deployment\recoveryPublic.key" -Region ${AWS::Region}
+              waitAfterCompletion: '0'
+        configSignal:
+          commands:
+            0-signalCompletion:
+              command: !Sub >
+                cfn-signal.exe -e %ERRORLEVEL% --stack ${AWS::StackId} --resource VaultMachine --region ${AWS::Region}
+              waitAfterCompletion: 0
+    CreationPolicy:
+      ResourceSignal:
+        Timeout: PT15M
+  VaultDRMachine:
+    Type: 'AWS::EC2::Instance'
+    Properties:
+      Tags:
+        - Key: Name
+          Value: !Sub '${VaultInstanceName} DR'
+      SecurityGroupIds: !Ref VaultInstanceSecurityGroups
+      SubnetId: !Ref DRInstanceSubnetId
+      ImageId: !FindInMap 
+        - RegionMap
+        - !Ref 'AWS::Region'
+        - Vault
+      InstanceType: !Ref VaultInstanceType
+      UserData: !Base64 
+        'Fn::Sub': >-
+          <script> cfn-init.exe -v -s ${AWS::StackId} -r VaultDRMachine
+          --configsets ascending --region ${AWS::Region} </script>
+      KeyName: !Ref KeyName
+      IamInstanceProfile: !Ref VaultInstancesProfile
+    Metadata:
+      'AWS::CloudFormation::Init':
+        configSets:
+          ascending:
+            - configSSMAndHostname
```

Comment:
same here, since we don't change the hostname we should alter the name

Reply:
changed